alarm/kodi-rbp-git/PKGBUILD

```diff
@@ -44,10 +44,10 @@ install:' -i tools/EventClients/Makefile.in
 }
 
 build() {
-  cd "${srcdir}/xbmc-${_commit}"
+  cd "${srcdir}/xbmc-${_tag}"
 
   # Bootstrapping
-  MAKEFLAGS=-j1 ./bootstrap
+  MAKEFLAGS=-j5 ./bootstrap
```

Comment:
Please just update the code and squash commits in the future. No need for
repetitious PR :)